### DISCUSSION TITLE
Items cannot be stored in the bank

### DISCUSSION THREAD

**Discussion by EspectroW94** [AscensionGameDev/Intersect-Engine#2430] [General]
Items cannot be stored in the bank

video:

https://github.com/user-attachments/assets/31934854-751b-48dd-beb2-f747078370e3

**Comment by EspectroW94** [comment]
Version 0.8.0.245

**Comment by Arufonsu** [comment]
hello, you are supposed to report bugs [here ](https://github.com/AscensionGameDev/Intersect-Engine/issues) instead of opening a discussion. closing this as "dupe" since @WeylonSantana opened https://github.com/AscensionGameDev/Intersect-Engine/issues/2431
